### ISSUE TITLE
Train model issue

### ISSUE THREAD

**Issue by BerindeieAlexandru** [ArnabKumarRoy02/ResEmoteNet#16]
Hello, I am having some issue trying to reproduce the results presented in paper. I am trying to train the model on FER2013 dataset and I am getting only around 60% accuracy. What could be the issue? Using the checkpoint provided I get almost the accuracy presented. Did you use extra training data or fine-tuned the model? Thank you!

**Comment by ArnabKumarRoy02** [comment]
Hey @BerindeieAlexandru, I have given the link for the FER2013 dataset that was used in this project in #12. You can check it out.

Also, I'm closing this issue, feel free to re-open it if you get stuck anywhere.
